p2719.md

```diff
@@ -34,20 +37,22 @@ process performed by the compiler before and after this paper:
 // user writes:
 new (args...) T(...)
 
-// compiler checks (in order):
-T::operator new(sizeof(T), args...)
-::operator new(sizeof(T), args...)
+// compiler checks:
+{{DeclScope = (T specifies operator new) ? T : Global}}
+DeclSpace::operator new(sizeof(T), align_val_t(alignof(T)), args...) // (1)
+DeclSpace::operator new(sizeof(T), args...)  // (2)
```

Comment:
I understand that you're trying to be precise here by treating `align_val_t` consistently with what happens in reality. But since this is just a comparison table, I'd rather stay a bit hand-wavy but more easy to grok at first sight. We can explain the detailed lookup + overload resolution later in the paper.